source/gloperate/include/gloperate/pipeline/Stage.h

```diff
@@ -783,6 +836,13 @@ class GLOPERATE_API Stage : public cppexpose::Object
     Environment * m_environment;    ///< Gloperate environment to which the stage belongs
     bool          m_alwaysProcess;  ///< Is the stage always processed?
 
+	std::vector<std::uint64_t>  m_cpuTimes;
+	std::vector<std::uint64_t>  m_gpuTimes;
+	unsigned int                m_measurementCycle;
+	unsigned int                m_measurementHistorySize;
+	unsigned int                m_queryID[2][2];
```

Comment:
Please change the type of `m_queryID` to be more safe and C++-idiomatic (e.g., `std::array<unsigned int, 4>`).